doc/source/whatsnew/v0.23.0.txt

```diff
@@ -406,6 +403,18 @@ Indexing
 - Bug in :func:`MultiIndex.set_labels` which would cause casting (and potentially clipping) of the new labels if the ``level`` argument is not 0 or a list like [0, 1, ... ]  (:issue:`19057`)
 - Bug in ``str.extractall`` when there were no matches empty :class:`Index` was returned instead of appropriate :class:`MultiIndex` (:issue:`19034`)
 
+MultiIndex
+^^^^^^^^^^
+
+- Bug in :func:`MultiIndex.get_level_values` which would return an invalid index on a level of ints with missing values (:issue:`17924`)
+- Bug in :func:`MultiIndex.remove_unused_levels` which would fill nan values (:issue:`18417`)
+- Bug in :func:`MultiIndex.from_tuples`` which would fail to take zipped tuples in python3 (:issue:`18434`)
```

Comment:
are all of the issues in the PR top covered here?

Reply:
Yes, except #18519 itself